server/src/test/java/io/crate/integrationtests/UppercaseSchemaNameIntegrationTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.integrationtests;
+
+import static io.crate.testing.TestingHelpers.printedTable;
+import static org.assertj.core.api.Assertions.assertThat;
+import static io.crate.testing.Asserts.assertThatThrownBy;
+
+import java.util.List;
+
+import org.elasticsearch.test.IntegTestCase;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.crate.expression.udf.UserDefinedFunctionService;
+import io.crate.expression.udf.UserDefinedFunctionsMetadata;
+import io.crate.metadata.RelationName;
+import io.crate.metadata.view.ViewMetadata;
+import io.crate.metadata.view.ViewsMetadata;
+import io.crate.types.StringType;
+
+public class UppercaseSchemaNameIntegrationTest extends IntegTestCase {
+
+    @Override
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        var dummyLang = new UserDefinedFunctionsIntegrationTest.DummyLang();
+        Iterable<UserDefinedFunctionService> udfServices = internalCluster().getInstances(UserDefinedFunctionService.class);
+        for (UserDefinedFunctionService udfService : udfServices) {
+            udfService.registerLanguage(dummyLang);
+        }
+    }
+
+    @Test
+    public void test_using_schema_names_with_uppercases() {
```

Comment:
Since the majority of the changes that enable this is from `AstBuilder`, this test maybe unnecessary?